packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -42,6 +43,7 @@ function markupRenderOptions(override: MarkupToHtmlOptions = null): MarkupToHtml
 			},
 		},
 		replaceResourceInternalToExternalLinks: true,
+		allowedFilePrefixes: [Setting.value('resourceDir')],
```

Comment:
**Note:** This currently allows `file://` URLs that point to the resource directory in all rendering done by the rich text editor.

I'm currently checking to see if this can be restricted to one of the several usages of `markupRenderOptions`.